resources/howtos/digitalocean_droplet.md

```diff
@@ -0,0 +1,109 @@
+# How to deploy a MeiliSearch instance on DigitalOcean
+
+## Create an out-of-the-box MeiliSearch
+
+### 1. Create a new droplet
+
+In any DigitalOcean page, when you are logged in, you will find a menu in the upper-right corner. Click on Create -> Droplets
+
+![Create droplet](../../public/image/digitalocean/01.create.png)  
+
+### 2. Select MeiliSearch snapshot
+
+By default DigitalOcean will show you the "distributions" tab. Select the "Marketplace" tab and search for "meili". Select it.
+
+![Marketplace](../../public/image/digitalocean/02.marketplace.png)  
+
+### 3. Select your plan
+
+Select your plan, starting at $5 (click on 'See all plans for more options). Memory optimized options will give you better results for a production environment on big datasets.
```

Comment:
```suggestion
Select your plan. Plans start at $5 (click on 'See all plans for more options). Memory-optimized options will give you better results for a production environment on big datasets.
```

Reply:
There is a missing single quote (`'`) but I don't know where -> 'See all plans

Reply:
Thanks!